conf/messages

```diff
@@ -1090,44 +1093,44 @@ tradingpremises.subheading = Trading premises
+estateagentbusiness.penalisedunderestateagentsact.lbl.detailslbl = Describe the penalty received under the Estate Agents Act 1979 and when it occurred
+estateagentbusiness.penalisedbyprofessional.title = Has the business, or anyone in the business, ever been penalised by a professional body for estate agency business activities?
 estateagentbusiness.penalisedbyprofessional.heading = Has the business, or anyone in the business, ever been penalised by a professional body?
-estateagentbusiness.penalisedbyprofessional.subtitle = The penalty should relate to estate agency business activities. Examples of a penalty include:
+estateagentbusiness.penalisedbyprofessional.subtitle = Examples of a penalty include:
 estateagentbusiness.penalisedbyprofessional.line_1 = a fine
 estateagentbusiness.penalisedbyprofessional.line_2 = a warning
 estateagentbusiness.penalisedbyprofessional.line_3 = suspension
 estateagentbusiness.penalisedbyprofessional.line_4 = expulsion
 estateagentbusiness.penalisedbyprofessional.line_5 = special terms or conditions imposed on your business
-estateagentbusiness.penalisedbyprofessional.lbl.detailslbl = Give a brief overview of the penalty and when it occurred.
+estateagentbusiness.penalisedbyprofessional.lbl.detailslbl = Describe the penalty received by  a professional body, and when it occurred
 estateagentbusiness.services.title = Which services does your business provide?
-estateagentbusiness.service.lbl.01 = Residential estate agency services
-estateagentbusiness.service.lbl.02 = Commercial estate agency services
-estateagentbusiness.service.lbl.03 = Auctioneering
-estateagentbusiness.service.lbl.04 = Relocation agent services
-estateagentbusiness.service.lbl.05 = Business transfer agent services
-estateagentbusiness.service.lbl.06 = Asset management
-estateagentbusiness.service.lbl.07 = Land management
-estateagentbusiness.service.lbl.08 = Development company services
+estateagentbusiness.service.lbl.08 = Residential
+estateagentbusiness.service.lbl.04 = Commercial
+estateagentbusiness.service.lbl.02 = Auctioneering
+estateagentbusiness.service.lbl.07 = Relocation
+estateagentbusiness.service.lbl.03 = Business transfer
+estateagentbusiness.service.lbl.01 = Asset management
+estateagentbusiness.service.lbl.06 = Land management
+estateagentbusiness.service.lbl.05 = Development company
 estateagentbusiness.service.lbl.09 = Social housing provision
 estateagentbusiness.redress.lbl = Redress scheme:
-estateagentbusiness.redress.lbl.01 = The Property Ombudsman Limited
-estateagentbusiness.redress.lbl.02 = Ombudsman Services
-estateagentbusiness.redress.lbl.03 = Property Redress Scheme
+estateagentbusiness.redress.lbl.01 = Ombudsman Services
+estateagentbusiness.redress.lbl.02 = Property Redress Scheme
+estateagentbusiness.redress.lbl.03 = The Property Ombudsman Limited
 estateagentbusiness.redress.lbl.04 = Other
```

Comment:
I'm not sure you should change the label's texts... let's say you selected estateagentbusiness.redress.lbl.01 before changes, value 01 was mapped to The Property Ombudsman Limited, you are coming back to your application after changes are deployed, then 01 is mapped to Ombudsman Services... :( unless you have necessary changes in the model mapping

Reply:
Will fix in 4928